openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -739,6 +743,23 @@ def validate_file_volumes_count(self, h_node, h_server, node_ip):
             "Failed to verify volume count Expected:'{}', Actual:'{}'".format(
                 len(h_volume_list['volumes']), len(vol_list)))
 
+    def match_volume_by_prefix(self, prefix):
+        """Match PVC, PV, Heketi and Gluster volume
```

Comment:
Keep the heketi and gluster as in lower case.

Reply:
@adityaramteke why in lower case ?

Reply:
Always lowercase program/project when the word stands alone. Heketi and Gluster are also names of an organization. I agree to keep them in capitalize.